flybywire-aircraft-a320-neo/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "dependencies": [],
+  "content_type": "AIRCRAFT",
+  "title": "A32NX",
+  "manufacturer": "Airbus",
+  "creator": "FlyByWire Simulations",
+  "package_version": "0.6.0",
+  "minimum_game_version": "1.14.6",
+  "release_notes": {
+    "neutral": {
+      "LastUpdate": "VERSION 0.6.0 RELEASED APRIL 6, 2021\\nInitial Release onto Marketplace",
```

Comment:
Should probably change this, since 0.6.0 hasn't even been released yet

Reply:
I thought the first version going onto the marketplace was V0.6.0

Reply:
Yes, however we are not planning on releasing it on April 6th. Also, this file can be deleted. Similar to layout.json, we also generate ours. I do recommend editing manifest-base.json in the root of the project to add the `LastUpdate` field

Reply:
LastUpdate exists in the manufest-base.json, it is just not populated. We could populate this however would be best when we know the definitive release date. I could add `"LastUpdate": "VERSION 0.6.0 RELEASED APRIL 6, 2021\\nInitial Release onto Marketplace",` as a placeholder if we wanted to ensure compatibility with the Marketplace?

Reply:
Hmm. Maybe we could do that without the released part. I don't think it requires a release date

Reply:
This is how it shows on the 'Version History'

![image](https://user-images.githubusercontent.com/50173847/113643025-26bc0e80-9679-11eb-8d39-fea348a97e90.png)

Reply:
This is how it looks without the date. 
![image](https://user-images.githubusercontent.com/50173847/113643163-8e725980-9679-11eb-99c2-d4dee0b15de2.png)

Reply:
Maybe `VERSION 0.6.0\\nInitial Release onto Marketplace`

Reply:
LGTM
![image](https://user-images.githubusercontent.com/50173847/113643232-c4174280-9679-11eb-80e2-204c486c330b.png)

Reply:
Yep, that looks good

Reply:
Don't forget to delete this file